spreadsheets.js

```diff
@@ -0,0 +1,215 @@
+var moment = require("moment");
+var fuzzy_match = require("./fuzzy_match");
+var GoogleSpreadsheet = require("google-spreadsheet");
+
+var ISO_TUESDAY = 2;
+var DATE_FORMATS = [
+    "YYYYMMDD HHmm",
+    "YYYYMMDD Hmm",
+    "HHmm DDMMYYYY"
+];
+var WORDS_TO_IGNORE = [
+    "",
+    "-",
+    "–",
+    "at",
+    "on",
+    "gmt",
+    "rescheduled",
+    "reschedule",
+    "reschedule:",
+    "to",
+    "v",
+    "vs",
+    "vs.",
+    "versus",
+    "sunday",
+    "monday",
+    "tuesday",
+    "wednesday",
+    "thursday",
+    "friday",
+    "saturday",
+    "sun",
+    "mon",
+    "tues",
+    "wed",
+    "thu",
+    "thur",
+    "thurs",
+    "fri",
+    "sat",
+
+    // TODO: yes, this is oddly specific - but unless he changes his username ....
+    "steveharwell"
```

Comment:
For what it's worth, it's extremely easy to change your username with slack.  I would prefer he used his lichess username on slack.  It makes loads of stuff easier.